va-gov/layouts/home.html

```diff
@@ -92,7 +92,7 @@ <h3><i class="icon-large fa far fa-id-card"></i>Records</h3>
 
       <div class="usa-width-one-third">
         <button class="homepage-button secondary-dark">
-          <a href=https://www.veteranscrisisline.net/>
+          <a data-show="#modal-crisisline" class="va-overlay-trigger" href="javascript:void(0)">
```

Comment:
This will trigger the Crisis Line dialog. The `javascript:void(0)` is necessary for the link checker to okay it.